docs/pages/index.mdx

```diff
@@ -2,39 +2,134 @@ import ImageRow from "@/components/ImageRow";
+
+In many cases, but not all, the Director’s hostname can also be used as the federation’s hostname because it provides the same set of metadata. The OSDF breaks these two endpoints apart by providing federation metadata at osg-htc.org, which is duplicated within the OSDF Director. This allows the OSDF Director to use a subdomain of osg-htc.org, which is otherwise a statically-hosted informational website.
+
+All Origins and Staging Servers in a federation send periodic advertisements to the discovered Director at a default interval of 10 minutes to let it know where they can be accessed, which namespace(s) they provide, and any information pertaining to data access policies (such as authorization schemes). In this way, the Director is the only service that has a nearly real-time view of all the Origins and Staging devices in the federation -- if an Origin or Staging Server fails to re-advertise after the required period, it is assumed to be offline until another advertisement is received, and the Director will stop sending clients to that location.
+
+#### Registry Service
+
+Whenever a new Origin or Staging device is created and added to a federation, its first step is to register itself with the *Registry*, which acts as the federation's locus of trust. In the case of Origins, the process of registration entails sending the Registry the namespace prefix the Origin exports, along with the Origin's public key and a variety of other bookkeeping information. After the Registry and the Origin have performed a handshake that proves the Origin owns the corresponding private key, the Registry stores the information in a persistent database.
+
+This process serves two purposes -- first, whenever the Origin re-advertises with the federation's Director, the Director can verify the authenticity of those advertisements through public/private key asymmetric cryptography by looking at the Registry's stored public key for that Origin and namespace. Second, the Registry's persistent database prevents other Origins from registering namespaces under an already-registered namespace without first proving they're allowed to do so by the namespace owner (i.e. the entity that possesses the appropriate private key).
+
+## Making Bytes Accessible and Moving Them -- A First Look Under The Hood
```

Comment:
I forget the conclusion we draw during the RCF meeting. Do we want to split the glossary from the arch sections or just keep them on the same spot? Looking at the meeting notes, it seems that we want to have overview and core concepts as two separate pages:
https://docs.google.com/document/d/11knkvPU78eBvRZjcoYGzKV2F5y_Sp76qIeginR5NkIk/edit
```
About Pelican
– Core Concepts

```